### ISSUE TITLE
formatter: error when using assert function

### ISSUE THREAD

**Issue by verillious** [Scony/godot-gdscript-toolkit#122]
Hey - I'm getting the following error using gdformat:

```
> gdformat ./globals/transition/Transition.gd
exception during formatting of ./globals/transition/Transition.gd
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/bin/gdformat", line 33, in <module>
    sys.exit(load_entry_point('gdtoolkit==3.2.8', 'console_scripts', 'gdformat')())
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/gdtoolkit/formatter/__main__.py", line 156, in main
    raise e
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/gdtoolkit/formatter/__main__.py", line 145, in main
    formatted_code = format_code(
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/gdtoolkit/formatter/formatter.py", line 48, in format_code
    formatted_lines, _ = format_block(
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/gdtoolkit/formatter/block.py", line 38, in format_block
    lines, previously_processed_line_number = statement_formatter(
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/gdtoolkit/formatter/class_statement.py", line 53, in format_class_statement
    return handlers[statement.data](statement, context)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/gdtoolkit/formatter/class_statement.py", line 268, in _format_func_statement
    func_lines, last_processed_line_no = format_block(
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/gdtoolkit/formatter/block.py", line 38, in format_block
    lines, previously_processed_line_number = statement_formatter(
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/gdtoolkit/formatter/function_statement.py", line 27, in format_func_statement
    return handlers[statement.data](statement, context)
KeyError: 'legacy_assert_stmt'
```

(the file in question)

```
extends CanvasLayer

signal scene_changed

var new_scene
var current_scene

onready var music_player = $AudioStreamPlayer
onready var animation_player = $AnimationPlayer
onready var color_rect = $Control/ColorRect


func change_scene(path: String, delay: float = 0.5) -> void:
	yield(get_tree().create_timer(delay), "timeout")

	animation_player.play("Fade")
	yield(animation_player, "animation_finished")

	var changed_scene = get_tree().change_scene(path)
	assert(changed_scene == OK)

	animation_player.play_backwards("Fade")
	yield(animation_player, "animation_finished")

	emit_signal("scene_changed")
```

The file is formatted fine if I remove the assert, but it looks fine to me so I'm not sure what the issue is

**Comment by Scony** [comment]
This has been fixed in ad6a83d2a070e788d1f1f7b06a5f2e40e48e6b20 and thus will be a part of next release.

You can use it already by installing `gdtoolkit` from `master`:
```
python -m pip install git+https://github.com/Scony/godot-gdscript-toolkit.git
```

**Comment by verillious** [comment]
Brilliant, thank you @Scony !
